.travis.yml

```diff
@@ -24,6 +24,8 @@ jobs:
     - stage: test
       name: "Build and test"
       script: travis_retry mvn test
+      addons:
```

Comment:
Can this be made as a separate job, independent of the "Build and test"?
Might be for testing reasons, but this doesn't seem to be restricted to master and support branches only.